frontend-new/src/components/layout/Header.vue

```diff
@@ -123,27 +128,62 @@ authLog("render with user " + authStore.user?.name);
           >
             {{ navBarLink.label }}
           </router-link>
-        </nav>
+        </div>
       </div>
 
-      <div class="login-buttons flex gap-2 items-center">
-        <button class="flex mr-2" @click="theme.toggleDarkMode()">
+      <!-- Right side items -->
+      <div class="flex items-center gap-2">
+        <div v-if="authStore.user" class="flex items-center">
+          <DropdownButton name="Create">
+            <DropdownItem>Create project</DropdownItem>
+            <DropdownItem>Create organization</DropdownItem>
+          </DropdownButton>
```

Comment:
Should this be localised or do we not care about that yet for the new frontend?

Reply:
they should, same with the dropdowns, there are even constants for them already iirc